docker/Dockerfile

```diff
@@ -400,36 +404,15 @@ RUN install-from-source https://github.com/nanopb/nanopb/archive/refs/tags/0.4.9
 ### ADD NEW PROGRAMS/LIBRARIES HERE ###
 #######################################
 
-# Install python libraries
```

Comment:
Is poetry used inside the container now for python deps?

Reply:
Sort of, for anything that is used for our bids it is. Anything else is just installed on demand before the install using pip (which is inside a venv in /usr/local)